helm/clickhouse/values.yaml

```diff
@@ -298,4 +298,5 @@ pluralRunbook:
   enabled: true
 
 pre-delete-installation:
```

Comment:
Change to `preDeleteHook` or `preDeleteInstallationHook` since `-` can cause all sorts of problems in Go templating.